pkg/truncindex/truncindex.go

```diff
@@ -10,9 +10,8 @@ import (
 )
 
 var (
-	// ErrNoID is thrown when attempting to use empty prefixes
-	ErrNoID        = errors.New("prefix can't be empty")
-	errDuplicateID = errors.New("multiple IDs were found")
+	ErrEmptyPrefix     = errors.New("Prefix can't be empty")
+	ErrAmbiguousPrefix = errors.New("Too many containers found with provided prefix")
```

Comment:
Because this is a generic package you should not have the word `contianers` in the error message.  It mostly deals with strings as this is an index.

Can you change this to be something more generic for this package?